texar/torch/data/data/dataset_utils.py

```diff
@@ -57,9 +57,33 @@ def connect_name(lhs_name, rhs_name):
 
 
 class Batch:
-    r"""Wrapper over Python dictionaries representing a batch. This provides a
-    common interface with :class:`~texar.torch.data.data.dataset_utils.Batch`
-    that allows accessing via attributes.
+    r"""Wrapper over Python dictionaries representing a batch. It provides a
+    dictionary-like interface to access its fields. This class can be used in
+    the followed way
+
+    .. code-block:: python
+
+            hparams = {
```

Comment:
This is over-indented by 4 spaces.

Reply:
This class also supports accessing fields via the "dot" notation, i.e. `batch.text_ids`. Might as well demonstrate that in this example. I also feel this example contained irrelevant parts, for instance, we don't really need a `model` here, we can just iterate over the iterator and do nothing.

Reply:
Made the change in aa75023